client/lobby/SelectionTab.cpp

```diff
@@ -443,13 +448,17 @@ void SelectionTab::filter(int size, bool selectFirst)
 			std::shared_ptr<ElementInfo> folder = std::make_shared<ElementInfo>();
 			folder->isFolder = true;
 			folder->folderName = folderName;
+			folder->displayName = TextOperations::shortenUnicodeString(folder->folderName, folderNameMaxDisplayLength);
```

Comment:
Don't you want to add ... in the end of string if truncated?

Reply:
done